pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go

```diff
@@ -155,6 +154,16 @@ SET enable_experimental_stream_replication = true;
 	destSQL.ExpectErr(t, "pq: either old tenant ID 10 or the new tenant ID 1 cannot be system tenant",
 		`RESTORE TENANT 10 FROM REPLICATION STREAM FROM $1 AS OF SYSTEM TIME `+startTime+` AS TENANT `+fmt.Sprintf("%d", roachpb.SystemTenantID.ToUint64()),
 		pgURL.String())
+
+	existingTenantID := uint64(10)
+	_, existingTenantConn := serverutils.StartTenant(t, hDest.SysServer, base.TestTenantArgs{TenantID: roachpb.MakeTenantID(existingTenantID)})
+	defer func() {
+		require.NoError(t, existingTenantConn.Close())
+	}()
+	destSQL.ExpectErr(t, fmt.Sprintf("pq: tenant with id %d already exists", existingTenantID),
+		`RESTORE TENANT 10 FROM REPLICATION STREAM FROM $1 AS OF SYSTEM TIME `+startTime+` AS TENANT `+fmt.Sprintf("%d", existingTenantID),
+		pgURL.String())
```

Comment:
Do we need to StartTenant here or could we get away with `SELECT crdb_internal.create_tenant($1)`?  If you don't mind, I also think we should move this to a different test function.

Reply:
Good point, switched over to `create_tenant` and moved the two `ExpectErr`'s to their own `TestTenantStreamingCreationErrors`